helm/aws-operator/templates/_helpers.tpl

```diff
@@ -2,34 +2,34 @@
 {{/*
 Expand the name of the chart.
 */}}
-{{- define "aws-operator.name" -}}
+{{- define "name" -}}
 {{- .Chart.Name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
 {{/*
 Create chart name and version as used by the chart label.
 */}}
-{{- define "aws-operator.chart" -}}
+{{- define "chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
 {{/*
 Common labels
 */}}
-{{- define "aws-operator.labels" -}}
-app: {{ include "aws-operator.name" . | quote }}
-{{ include "aws-operator.selectorLabels" . }}
+{{- define "labels" -}}
+app: {{ include "name" . | quote }}
+{{ include "selectorLabels" . }}
 app.giantswarm.io/branch: {{ .Values.project.branch | quote }}
 app.giantswarm.io/commit: {{ .Values.project.commit | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-helm.sh/chart: {{ include "aws-operator.chart" . | quote }}
+helm.sh/chart: {{ include "chart" . | quote }}
 {{- end -}}
 
 {{/*
 Selector labels
 */}}
-{{- define "aws-operator.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "aws-operator.name" . | quote }}
+{{- define "selectorLabels" -}}
```

Comment:
WDYT about this to make alphabet gods happy (also variable grouping here would make more sense to me)?
 
```suggestion
{{- define "labels.selector" -}}
```

Reply:
This doesn't work as we already use `labels` as a string.

Reply:
We could do `labels.common` and `labels.selector`

Reply:
Sounds good to me. I didn't know that a variable can't be a prefix of another one.

Reply:
done